pkg/cli/clients/management.go

```diff
@@ -373,7 +373,7 @@ func (amc *UCPApplicationsManagementClient) DeleteApplication(ctx context.Contex
 		resource := resource
 		g.Go(func() error {
 			_, err := amc.DeleteResource(groupCtx, *resource.Type, *resource.ID)
-			if err != nil {
+			if err != nil && !clientv2.Is404Error(err) {
```

Comment:
This was missing before in the delete application flow. If the app was not found, we were always returning an error. I don't think this is the right approach. Resource not existing means resource has already been deleted.